vendor/bat-native-ads/src/bat/ads/internal/ad_events/ad_events.cc

```diff
@@ -69,11 +69,9 @@ void RecordAdEvent(const AdEventInfo& ad_event) {
   const std::string confirmation_type_as_string =
       std::string(ad_event.confirmation_type);
 
-  const base::Time time = base::Time::Now();
-  const uint64_t timestamp = static_cast<uint64_t>(time.ToDoubleT());
```

Comment:
When rebuilding ad events after a browser restart all timestamps were incorrectly set to the current timestamp, but should have been set to the ad event timestamp